lib/puppet.rb

```diff
@@ -36,6 +37,10 @@ module Puppet
   require 'puppet/environments'
 
   class << self
+    include FastGettext::Translation
```

Comment:
I think that `include FastGettext::Translation` should define the `#_()` translation method, while `GettextSetup.initialize` does the same thing on `Object` (https://github.com/puppetlabs/gettext-setup-gem/blob/master/lib/gettext-setup/gettext_setup.rb#L18), so we should only need to do one of these.